parts/typeform-popup.php

```diff
@@ -0,0 +1,13 @@
+<?php
+    $style = ($style) ? $style: 'link';
+	$type = ($type == 'drawer') ? 2: 1;
```

Comment:
Could you avoid the reassignment of the variable here?

Reply:
That's true, I didn't refactor this.